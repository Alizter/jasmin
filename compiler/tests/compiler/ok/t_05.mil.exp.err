Parsing tests/compiler/ok/t_05.mil as MIL

Processed file tests/compiler/ok/t_05.mil
