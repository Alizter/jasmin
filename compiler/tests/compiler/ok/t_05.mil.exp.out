merging basic blocks in all function(s)
   REMOVED_TIME
>> input:
fn foo1(x.1 : stack u64, y.2 : stack u64, z.3 : reg bool) -> stack u64 {
  w.4 : stack u64
  // START
  (w.4 : stack u64) = (x.1 : stack u64);
  (w.4 : stack u64) = (w.4 : stack u64) + (x.1 : stack u64);
  // END
  // START
  if $(w.4 : stack u64 = 5) {
    // START
    (z.3 : reg bool), (x.1 : stack u64) = (x.1 : stack u64) + (w.4 : stack u64);
    (z.3 : reg bool), (x.1 : stack u64) = (x.1 : stack u64) + (y.2 : stack u64);
    // END
  }
  // END
  return (x.1 : stack u64);
}

